umm-spec-lib/src/cmr/umm_spec/xml_to_umm_mappings/iso_smap.clj

```diff
@@ -119,7 +119,7 @@
   Category of each theme descriptive keyword to determine if it is a science keyword."
   [data-id-el sanitize?]
   (if-let [science-keywords (seq
-                              (->> (kws/parse-science-keywords data-id-el sanitize?)
+                              (->> (kws/parse-science-keywords data-id-el sanitize? true)
```

Comment:
It would be nice here if you defined a variable like smap? as true in this function and sent that instead of true, or put in a comment describing what true is.  I had to go to the parse-science-keywords function to figure out what true meant.

Reply:
Added comment